src/app/space/analyze/analyze-overview/analyze-overview.spec.ts

```diff
@@ -223,6 +109,6 @@ describe('AnalyzeOverviewComponent', () => {
     userService.loggedInUser.connect();
     this.detectChanges();
 
-    expect(this.testedDirective.isInOtherUsersSpace()).toBe(true);
+    expect(this.testedDirective.userOwnsSpace()).toBe(false);
```

Comment:
Please also add tests to exercise the not-logged-in case:

`if (this.context && this.loggedInUser) {`

So far you've tested the logic within this branch, where there is a context set and a user is logged in, and verified that the function works as expected if this user owns the space. There should also be tests where this branch is not entered, so `this.context` and/or `this.loggedInUser` are falsey.